src/sentry/rules/actions/base.py

```diff
@@ -1,10 +1,32 @@
 from __future__ import absolute_import, print_function
 
+import functools
+import logging
+import operator
+import six
+
+from django import forms
+
 from sentry.constants import ObjectStatus
 from sentry.models.integration import Integration
 from sentry.models import ExternalIssue, GroupLink
 from sentry.rules.base import RuleBase
 
+logger = logging.getLogger("sentry.rules")
+
+INTEGRATION_KEY = "integration"
+
+
+class NotifyServiceForm(forms.Form):
```

Comment:
@ceorourke can we call this `Integration NotifyServiceForm` instead? I think that's more specific.